node-tests/blueprints/component-test.js

```diff
@@ -324,6 +329,21 @@ describe('Blueprint: component', function () {
       );
     });
 
+    it('component foo --strict', function () {
+      return emberGenerateDestroy(['component', 'foo', '--strict'], (_file) => {
+        expect(_file('app/components/foo.gjs')).to.equal(templateTagContents);
+
+        expect(_file('tests/integration/components/foo-test.gjs')).to.equal(
```

Comment:
The test file isn't being created when generating a component in this test:

```
  288 passing (2m)
  1 failing

  1) Blueprint: component
       in app - octane
         component foo --strict:
     AssertionError: expected "tests/integration/components/foo-test.gjs" to exist

parent path "tests/integration/components" does not exist.
parent path "tests/integration" exists and contains:
- .gitkeep
```

When trying out the local version using `yarn link` it does however generate the test file correctly. Anything I'm missing here?